app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrations.jsx

```diff
@@ -56,6 +56,11 @@ export const Integrations = () => {
     setPlugin({});
   }, []);
 
+  const goToInitialPage = useCallback(() => {
```

Comment:
Be as kind as to explain the reason for using of useCallback hook here.

Reply:
fixed ☑